### ISSUE TITLE
Creating of custom browser profiles does not work

### ISSUE THREAD

**Issue by zenderol** [SpeedCurve-Metrics/SpeedCurve#72]
After creating a custom profile, I get a message that the creation was successful, but the created profile does not appear under the available browsers.

![custom profile](https://cloud.githubusercontent.com/assets/10632238/13748377/e17dd0f2-e9fc-11e5-86c8-3ea08b3ecd9b.png)

![bildschirmfoto 2016-03-14 um 15 52 50](https://cloud.githubusercontent.com/assets/10632238/13748385/e862ea10-e9fc-11e5-8738-8fdb2eb80265.png)

**Comment by zeman** [comment]
Fixed now.

**Comment by zenderol** [comment]
Unfortunately, I cannot confirm that the issue is fixed. Please have a look at the settings of our Team, the name is “Beta Welt”.

**Comment by zeman** [comment]
I've added a custom profile to your account called "Desktop Fibre". You've been caught out but a different issues we're working on. If none of your sites have "All Browsers" turned on then we don't have a site to add the custom browser to. Since "Beta WELT" had all the browsers selected I switched it to "All Browsers" and then was able to add the custom browser. We're working on some messaging in the UI to let people know they need at least one site with "All Browsers" turned on in order to add more browsers.
